### PR TITLE
Add scorum network

### PR DESCRIPTION
**Pull request by mikluke**

### CODE REVIEW

**Comment by HeesungB** [comment]
@mikluke @dDeedev Hello. 
Can you edit the file using the link below?
https://github.com/chainapsis/keplr-chain-registry#requirement-details

You can also use the command below to check the validation locally.
```
yarn validate cosmos/scorum.json
```

File: cosmos/scorum.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    },
+    {
+      "coinDenom": "GAS",
+      "coinMinimalDenom": "gas",
+      "coinDecimals": 0
+    }
+  ],
+  "feeCurrencies": [
+    {
+      "coinDenom": "gas",
```
Comment (HeesungB) [inline cosmos/scorum.json:42]: @mikluke Could you rename `coinDenom`?
The `coinDenom` is `GAS`  in currencies, but here is `gas`.

Comment (mikluke) [reply cosmos/scorum.json:42]: hi @HeesungB , done

**Comment by HeesungB** [comment]
@mikluke Could you check your node?
```
Failed to connect websocket to /websocket rpc
```

**Pull request merged**